commands.go

```diff
@@ -12,36 +12,36 @@ import (
 )
 
 // mergedTaskSetOpts returns a TaskSetOpt that applies the various filters
-// that should be exerted by the ctx and cmdLine
-func mergedTaskSetOpts(ctx, cmdLine CmdLine) TaskSetOpt {
+// that should be exerted by the ctx and query
+func mergedTaskSetOpts(ctx, query Query) TaskSetOpt {
```

Comment:
alternatively we could also make a Query.Merge(Query) method that returns a new merged query.

Reply:
I think an exported method w/ a unit test is preferable, since we probably want to decide on an official strategy for merging these objects.

Reply:
> alternatively we could also make a Query.Merge(Query) method that returns a new merged query.

You mean like this? https://github.com/naggie/dstask/blob/d687a0f9aec634a808cfa0df4aecb06ab4f57732/task.go#L66

That's how it was before the refactor; I am restoring it. 👍